### PR TITLE
Bulk import

### PR DESCRIPTION
**Pull request by justjustie**
Bulk import should be working now (Testing not possible)

### CODE REVIEW

File: Hallowspeak/Pages/Lexicon.razor [hunk 1]
```diff
@@ -84,12 +106,48 @@
...
+            int currentValueIndex = 0;
+            foreach (string key in keys)
+            {
+                try
+                {
+                    newItem.KeyValues.Add(key, new LexiconValue(values[currentValueIndex++]));
+                }
+                catch (IndexOutOfRangeException)
+                {
+                    // Not all columns filled
```
Comment (justjustie) [inline Hallowspeak/Pages/Lexicon.razor]: This is kinda janky...

**Comment by justjustie** [comment]
Also you can have Single Submit and Bulk Import open at the same time and close neither of them

File: Hallowspeak/Pages/Lexicon.razor [hunk 2]
```diff
@@ -7,8 +7,26 @@
...
     </div>
 </div>
+@if (BulkImport)
+{
+    <form style="padding-bottom: 1rem;">
+        <div class="form-row">
+            <div class="col">
+                <label for="bulk-input">Copy and paste from the Google Sheet here:</label>
+                <br />
+                <textarea id="bulk-input" value="@BulkInput" />
```
Comment (dylanpiera) [inline Hallowspeak/Pages/Lexicon.razor]:
```suggestion
                <textarea id="bulk-input" @bind=BulkInput />
```

File: Hallowspeak/Pages/Lexicon.razor [hunk 3]
```diff
@@ -63,8 +81,12 @@
 @code {
     public bool CreatingNew { get; set; } = false;
 
+    public bool BulkImport { get; set; } = false;
+
     private LexiconItem NewItem { get; set; } = null;
 
+    private string BulkInput { get; set; } = string.Empty;
```
Comment (dylanpiera) [inline Hallowspeak/Pages/Lexicon.razor:88]: I find it funny that you do string.Empty instead of "" 😛

**Review by dylanpiera — CHANGES_REQUESTED** [review]
I have to fix the Data that will crash in your current loop, and I'm not entirely sure if your for loop will work. So lemme take a look at that...

**Pull request merged**